packages/core/src/file-upload/file-upload-item-group.tsx

```diff
@@ -0,0 +1,27 @@
+import type { JSX, ValidComponent } from "solid-js";
+import {
+	type ElementOf,
+	Polymorphic,
+	type PolymorphicProps,
+} from "../polymorphic";
+
+export type FileUploadItemGroupCommonProps<
+	T extends HTMLElement = HTMLElement,
+> = {
+	id?: string;
+	style?: JSX.CSSProperties | string;
+};
+
+export type FileUploadItemGroupRootProps<
+	T extends ValidComponent | HTMLElement = HTMLElement,
+> = Partial<FileUploadItemGroupCommonProps<ElementOf<T>>>;
+
+export function FileUploadItemGroup<T extends ValidComponent = "ul">(
+	props: PolymorphicProps<T, FileUploadItemGroupRootProps<T>>,
+) {
+	return (
+		<Polymorphic as="ul" class="file-upload__item-group" {...props}>
```

Comment:
Classes shouldn't be applied by default.

Reply:
Removed